### PR TITLE
Requirements.txt for OpenVINO examples

### PR DESCRIPTION
**Pull request by alexsu52**
### Changes

Added requirements.txt for OpenVINO examples

### Reason for changes

Need to run examples

### Related tickets

N/A

### Tests

N/A

### CODE REVIEW

**Review by aiarkinx — APPROVED** [review]

File: examples/experimental/openvino/yolo_v5/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+yolov5
```
Comment (andrey-churkin) [inline examples/experimental/openvino/yolo_v5/requirements.txt:1]: Should we download the `yolov5` repo in this case?

File: examples/experimental/openvino/mobilenet_v2/requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+torchvision==0.10.1
```
[the remark this answers is not in the document]
Comment (alexsu52) [reply examples/experimental/openvino/mobilenet_v2/requirements.txt:1]: torchvision has torch as dependency (https://pypi.org/project/torchvision/)

File: examples/experimental/openvino/yolo_v5/requirements.txt (hunk 1 above)
Comment (alexsu52) [reply examples/experimental/openvino/yolo_v5/requirements.txt:1]: Could you suggest changes?

Comment (andrey-churkin) [reply examples/experimental/openvino/yolo_v5/requirements.txt:1]: I mean if we installed `yolov5` through the pip is it required to clone the yolov5 repo or not?

**Review by andrey-churkin — APPROVED** [review]

**Pull request merged**